packages/workers-shared/package.json

```diff
@@ -31,6 +31,8 @@
 		"check:type": "pnpm run check:type:tests && tsc",
 		"check:type:tests": "tsc -p ./asset-worker/tests/tsconfig.json",
 		"clean": "rimraf dist",
+		"deploy": "pnpm run deploy:router-worker && pnpm run deploy:asset-worker",
```

Comment:
Am I missing something but I don't see the `deploy:asset-worker` script definition here...

Reply:
yeah, we need to merge https://github.com/cloudflare/workers-sdk/pull/6542 first. I will handle that

Reply:
fixed